src/Microsoft.Health.Dicom.Core/Configs/CoreConfiguration.cs

```diff
@@ -3,24 +3,17 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-using Microsoft.Health.Api.Configuration;
-using Microsoft.Health.Api.Features.Cors;
 using Microsoft.Health.Core.Configs;
-using Microsoft.Health.Dicom.Core.Configs;
 
-namespace Microsoft.Health.Dicom.Api.Configs;
+namespace Microsoft.Health.Dicom.Core.Configs;
 
-public class DicomServerConfiguration : IApiConfiguration
+public class CoreConfiguration
```

Comment:
Totally open to another name here, landed on `CoreConfiguration` to map to `Microsoft.Health.Dicom.Core` much like `ApiConfiguration` now maps to `Microsoft.Health.Dicom.Api`